File: app/src/main/java/com/woods/blinkreader/fragment/PreferencesFragment.java
package com.woods.blinkreader.fragment;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import com.woods.blinkreader.R;

public class PreferencesFragment extends PreferenceFragmentCompat {
    public static final String TAG = PreferencesFragment.class.getName();

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.app_preferences, rootKey);
    }



}

File: app/src/main/java/com/woods/blinkreader/fragment/PreferencesFragment.kt
package com.woods.blinkreader.fragment

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.woods.blinkreader.R

class PreferencesFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.app_preferences, rootKey)
    }

    companion object {
        @JvmField
        val TAG: String = PreferencesFragment::class.java.name
    }
}